@here/olp-sdk-dataservice-read/lib/client/SummaryRequest.ts

```diff
@@ -17,7 +17,7 @@
  * License-Filename: LICENSE
  */
 
-import { HRN } from "./HRN";
+import { HRN } from "../HRN";
```

Comment:
Maybe it would be better to import it from "@here/olp-sdk-dataservice-read" ?

Reply:
updated